server/samples/ecs-params.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1
+task_definition:
+  task_execution_role: ecsTaskExecutionRole
```

Comment:
should add a note that this role should already exist or needs to be created as explained in the tutorial.